tt_eager/tt_numpy/functions.hpp

```diff
@@ -168,9 +173,9 @@ static Tensor ones_like(
 
 template <typename T>
 static Tensor arange(
-    int64_t start,
-    int64_t stop,
-    int64_t step,
+    const int64_t& start,
```

Comment:
Is this going to be a pattern in passing 64bit long types?

Reply:
No, I will remove the ref. I just copy pasted this from somewhere, so good catch